fuzzers/fuzz_anything/baby_no_std/src/main.rs

```diff
@@ -48,6 +48,7 @@ fn panic(_info: &PanicInfo) -> ! {
 
 /// Coverage map with explicit assignments due to the lack of instrumentation
 static mut SIGNALS: [u8; 16] = [0; 16];
+#[allow(static_mut_refs)]
```

Comment:
It's better to not use `static_mut_refs`. 
Ideally we should use this function instead to create the observer: https://github.com/AFLplusplus/LibAFL/blob/5324799935cc74762d9b945214a39f9916fcfdd6/fuzzers/structure_aware/baby_fuzzer_multi/src/main.rs#L85